scripts/train_test_plot.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
I'm not sure this particular file increases usability much. My personal preference is to keep everything in pure python as far as we can, and just to provide a command line argument in the README. But I'm happy with keeping this here.

Reply:
The use case was when we need to retrain large datasets, re-doing training + testing + plotting (it's very easy to just do `sh train_test_plot.sh` "michael ryan jdonati0904 ryan0904" testplot`.

I agree that this could just be a section from the README. This file is essentially me running things on my terminal and thinking "Ryan might be interested in knowing those commands if he needs to use my code, even if it's just to know how to use the python files."